redbot/cogs/economy/economy.py

```diff
@@ -922,16 +927,19 @@ async def slottime(self, ctx: commands.Context, seconds: int):
         await ctx.send(_("Cooldown is now {num} seconds.").format(num=seconds))
 
     @economyset.command()
-    async def paydaytime(self, ctx: commands.Context, seconds: int):
+    async def paydaytime(self, ctx: commands.Context, seconds: TimedeltaConverter):
         """Set the cooldown for the payday command.
 
         Example:
             - `[p]economyset paydaytime 86400`
+            - `[p]economyset paydaytime 1d`
 
         **Arguments**
 
         - `<seconds>` The new number of seconds to wait in between uses of payday. Default is 300.
+        - It is also a timedeltaconverter so `1d` is converted to 1 day
```

Comment:
This isn't really clear to the user, I would suggest sth like this:
```suggestion
        - `<seconds>` The new duration to wait in between uses of payday. Default is 5 minutes.
        Accepts: seconds, minutes, hours, days, weeks
```